api/src/main/java/org/pmiops/workbench/monitoring/StackdriverStatsExporterService.java

```diff
@@ -0,0 +1,56 @@
+package org.pmiops.workbench.monitoring;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.springframework.stereotype.Service;
+
+/**
+ * Simple wrapper service so we can mock out the initialization step (which depends directly on a
+ * public static method)
+ */
+@Service
+public class StackdriverStatsExporterService {
+  private static final Logger logger =
+      Logger.getLogger(StackdriverStatsExporterService.class.getName());
+  private static final String STACKDRIVER_CUSTOM_METRICS_DOMAIN_NAME = "custom.googleapis.com";
+
+  private boolean initialized;
+  private Provider<WorkbenchConfig> workbenchConfigProvider;
+
+  public StackdriverStatsExporterService(Provider<WorkbenchConfig> workbenchConfigProvider) {
+    this.workbenchConfigProvider = workbenchConfigProvider;
+    this.initialized = false;
+  }
+
+  /**
+   * Create and register the stats configuration on the stats exporter. This operation should only
+   * happen once, so we have an isInitialized guard for that.
+   */
+  public void createAndRegister() {
+    if (!initialized) {
+      try {
+        final StackdriverStatsConfiguration configuration =
+            StackdriverStatsConfiguration.builder()
+                .setMetricNamePrefix(buildMetricNamePrefix())
```

Comment:
Should this be stackdriver specific now? Or leave generic?

Reply:
The `StackdriverStatsConfiguration` object is theirs, not mine. It looks like OpenCensus stuffs all backend-specific things in their exporters. I don't think it's a good candidate for generic-hood until we know all the exporters will be the same shape. (I don't want to make an interface for existing third-party classes to support when I don't control any of those classes). Good idea, though.